verify-service/src/main/java/io/inji/verify/controller/VPSubmissionController.java

```diff
@@ -0,0 +1,61 @@
+package io.inji.verify.controller;
+
+import io.inji.verify.dto.authorizationRequest.StatusDto;
+import io.inji.verify.dto.submission.PresentationSubmissionDto;
+import io.inji.verify.dto.submission.ResponseAcknowledgementDto;
+import io.inji.verify.dto.submission.VPSubmissionDto;
+import io.inji.verify.dto.submission.VPTokenResultDto;
+import io.inji.verify.enums.SubmissionState;
+import io.inji.verify.shared.Constants;
+import io.inji.verify.spi.VerifiablePresentationRequestService;
+import io.inji.verify.spi.VerifiablePresentationSubmissionService;
+import io.inji.verify.singletons.GsonSingleton;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class VPSubmissionController {
+
+    @Autowired
+    VerifiablePresentationRequestService verifiablePresentationRequestService;
+
+    @Autowired
+    VerifiablePresentationSubmissionService verifiablePresentationSubmissionService;
+
+    @Autowired
+    GsonSingleton gsonSingleton;
+
+    @GetMapping(path = "/vp-result/{transactionId}")
```

Comment:
Does is this fit inside this controller ? 
As per the API categorization it was under VP request management, we should revisit this.

Reply:
I think we discussed to keep the result part as different controller right ? because of the exposure to internet vs protected endpoint